app/javascript/components/restaurant.jsx

```diff
@@ -69,6 +69,12 @@ class Restaurant extends React.Component {
                 </React.Fragment>
               }
             </div>
+            <div className='actions'>
```

Comment:
hey @aSadhankar - small change for semantics - I would maybe change the className of this to something like `website-link` or if we plan to add other things in the future something like `other-details`.

And then if you change that, if there is any CSS associated with `actions` that you want want to preserve for `other-details`, be sure to add it to `app/assets/stylesheets/components/restaurant.scss`